frappe_private_comment/overrides/whitelist/comment.py

```diff
@@ -82,3 +84,40 @@ def get_comment_visibility(name: str):
         return None
 
     return {"custom_visibility": doc.custom_visibility}
+
+
+@frappe.whitelist()
+def get_comment_replies(reference_doctype: str, comment_id: str):
+    """Get comment replies"""
+    return frappe.get_all(
+        "Comment",
+        filters={
+            "reference_doctype": reference_doctype,
+            "custom_reply_to": comment_id,
+        },
+        fields=["name", "content", "comment_by", "creation"],
+    )
+
+
+@frappe.whitelist()
+def get_all_replies(reference_doctype: str, reference_name: str):
+    """Get all replies for a comment in a structured format"""
+    replies = frappe.get_all(
+        "Comment",
+        filters={
+            "reference_doctype": reference_doctype,
+            "reference_name": reference_name,
+        },
+        fields=["name", "content", "comment_by", "creation", "custom_reply_to"],
+    )
+
+    # Create a dictionary to store the structured comments
+    structured_comments = dict()
+
+    for reply in replies:
+        if reply["custom_reply_to"] is None:
+            structured_comments.setdefault(reply["name"], [])
```

Comment:
Shouldn't we move to the next reply if `custom_reply_to` is None?

Reply:
Fixed